recipes/blpapi/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+export BLPAPI_ROOT="."
+$PYTHON setup.py install
+
+if [ -d "$BLPAPI_ROOT/Linux" ]; then
+   cp -v $BLPAPI_ROOT/Linux/lib*.so $PREFIX/lib/
+else
+   cp -v $BLPAPI_ROOT/Darwin/lib*.so $PREFIX/lib/
```

Comment:
it would be good if the script installation of the project already do that

Reply:
For this comment and your comment about related to the `License.txt` file when you are referring to the project you mean `blpapi`?